src/vs/workbench/common/editor/editorInput.ts

```diff
@@ -124,6 +125,10 @@ export abstract class EditorInput extends AbstractEditorInput {
 		return (this.capabilities & capability) !== 0;
 	}
 
+	getReadonlyMessage(): IMarkdownString | undefined {
```

Comment:
I would suggest a `isReadonly` method that by default checks on the `EditorInputCapabilities.Readonly`.

Reply:
@bpasero I'm not sure I'm understanding correctly. In this abstract class I don't think we have enough information to return a good message for readonly files.

Reply:
Yeah I was suggesting to return a `boolean` based on the `EditorInputCapabilities` and then subclasses can further tweak.

Reply:
Done, thank you for clarifying!

Reply:
This still seems pending.

Reply:
Started looking into something else and didn't push my changes. Pushed now.